docs/src/pages/components/buttons/buttons.md

```diff
@@ -180,3 +180,13 @@ However:
 ```
 
 This has the advantage of supporting any element, for instance, a link `<a>` element.
+
+## Unstyled
```

Comment:
Nit, let's make the description here the same as for the other unstyled components. (Why people would use it)